src/Containers/Artist/style.module.scss

```diff
@@ -0,0 +1,14 @@
+@import "../../common/util.scss";
+
+
+.container {
+  margin-bottom: 7vh;
+  @include portrait {
+    margin-left: var(--page-gutter);
```

Comment:
This makes the whole page uncentered?

Reply:
thanks for catching that. yeah that's copy pasta from
```
  @include medium {
    margin-left: var(--page-gutter);
    margin-bottom: 95px;
  }
```